analysers/experiments/executionTime.py

```diff
@@ -0,0 +1,71 @@
+import sys
+import json
+import io
+import gzip
+import uuid
+import math
+
+from datetime import timedelta
+
+import scipy.integrate as integrate
+import scipy.special as special
+import numpy as np
+
+from pyspark_cassandra import CassandraSparkContext
+from pyspark_cassandra import RowFormat
+from pyspark import SparkConf
+
+def createQuery(dataRDD, experimentID):
+    from commons import computeMode, computeMetrics
+    
+    queries = []
+    
+    processes = dataRDD.map(lambda a: a["process_definition_id"]).distinct().collect()
+    
+    for process in processes:
+        mode = computeMode(dataRDD.filter(lambda a: a["process_definition_id"] == process).map(lambda r: (r['execution_time'], 1)))
+    
+        data = dataRDD.filter(lambda r: r['process_definition_id'] == process).map(lambda r: r['execution_time']).collect()
+         
+        metrics = computeMetrics(data)
+        
+        queries.append({"process_definition_id": process, "experiment_id":experimentID, "execution_time_mode":mode[0], "execution_time_mode_freq":mode[1], \
+                  "execution_time_mean":metrics["mean"], "execution_time_num_data_points":metrics["num_data_points"], \
+                  "execution_time_min":metrics["min"], "execution_time_max":metrics["max"], "execution_time_sd":metrics["sd"], \
+                  "execution_time_q1":metrics["q1"], "execution_time_q2":metrics["q2"], "execution_time_q3":metrics["q3"], "execution_time_p95":metrics["p95"], \
+                  "execution_time_p90":metrics["p90"], "execution_time_p99":metrics["p99"], "execution_time_percentiles":metrics["percentiles"], \
+                  "execution_time_me":metrics["me"], "execution_time_ci095_min":metrics["ci095_min"], "execution_time_ci095_max":metrics["ci095_max"]})
+        
+    return queries
+
+def getAnalyserConf(SUTName):
+    from commons import getAnalyserConfiguration
+    return getAnalyserConfiguration(SUTName)
+
+def main():
```

Comment:
Where is the total execution time independent from the actual `process_definition_id`?